test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -125,16 +128,8 @@ object TestAspect extends TimeoutVariants {
     }
 
   /**
-   * Constructs an aspect that evaluates every test inside the context of a `Managed`.
-   */
-  def around[R0, E0](
-    before: ZIO[R0, E0, Any],
-    after: ZIO[R0, Nothing, Any]
-  ): PerTest[Nothing, R0, E0, Any, Nothing, Any] = around(before)(_ => after)
-
-  /**
-   * Constructs an aspect that evaluates every test inside the context of a `Managed` where the result of `before` can
-   * be used in `after`.
+   * Constructs an aspect that evaluates every test is evaluated between two effects, `before` and `after`,
```

Comment:
Constructs an aspect that evaluates every test between two effects, `before` and `after`, where the result of `before` can be used in `after`.